src/util/system.h

```diff
@@ -170,6 +171,11 @@ class ArgsManager
      */
     void WarnForSectionOnlyArgs();
```

Comment:
For completeness, could make this const as well?

Reply:
sure.

Reply:
done.